.gitignore

```diff
@@ -6,3 +6,4 @@
 venv/
 .pytest_cache/
 **__pycache__/
+db.sqlite3
```

Comment:
There is an extra space before `db.sqlite3`. It is recommended to remove the leading space to ensure it is correctly ignored by Git.